keyboards/tada68/config.h

```diff
@@ -66,15 +66,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* Underlight configuration
  */
 
-/*#define RGB_DI_PIN E2
+#define RGB_DI_PIN F5    // See the readme for wiring your ws2812's
 #define RGBLIGHT_ANIMATIONS
 #define RGBLED_NUM 2     // Number of LEDs
-#define RGBLIGHT_HUE_STEP 10
-#define RGBLIGHT_SAT_STEP 17
-#define RGBLIGHT_VAL_STEP 17*/
-//Disabled while I figure out a suitable pin for RGB support.
-//I've tried F5, D2, D3, and E2 but it's possible the end of my
-//strand is bad. New LEDs on order.
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
```

Comment:
Would you mind moving this to the keymap's config as well? It'll change per keymap :)